iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -461,44 +460,46 @@ public void appendFiles(org.apache.hadoop.hive.metastore.api.Table table, URI fr
   }
 
   @Override
-  public Map<String, String> getBasicStatistics(Partish partish) {
-    org.apache.hadoop.hive.ql.metadata.Table hmsTable = partish.getTable();
+  public Map<String, String> getBasicStatistics(org.apache.hadoop.hive.ql.metadata.Table hmsTable) {
```

Comment:
please don't change the method signature. it would be used in a follow-up PR

Reply:
ok. Revert this.